x-pack/plugins/stack_connectors/server/connector_types/lib/servicenow/service.test.ts

```diff
@@ -805,6 +876,220 @@ describe('ServiceNow service', () => {
     });
   });
 
+  describe('closeIncident', () => {
```

Comment:
Could you please add some tests where we check that we get the incident by correlation ID correctly (we call `requestMock` with the correct  URL etc)?

Reply:
Ok I just saw that you have them in `x-pack/plugins/stack_connectors/server/connector_types/servicenow_itsm/service.test.ts`. Should we move them here?